src/tests/system/tests/test_logging.py

```diff
@@ -14,147 +17,121 @@
         2. Check logs
     :expectedresults:
         1. SSSD failed to start
-        2. Correct error message is in log file
+        2. Logs contain error message
     :customerscenario: True
     """
     client.sssd.sssd["domains"] = "non_existing_domain"
-    assert (
-        client.sssd.start(debug_level=None, raise_on_error=False).rc != 0
-    ), "SSSD started successfully, which is not expected"
-    assert "id_provider is not set for domain [non_existing_domain]" in client.fs.read(client.sssd.logs.monitor)
+    assert client.sssd.start(debug_level=None, raise_on_error=False).rc != 0, "SSSD erroneously started!"
+    assert "No properly configured domains, fatal error!" in client.fs.read(
+        client.sssd.logs.monitor
+    ), "Domain is configured!"
 
 
 @pytest.mark.ticket(bz=1893159)
 @pytest.mark.topology(KnownTopology.LDAP)
-def test_logging__default_debug_level_nonexisting_ldap_server(client: Client):
+def test_logging__default_settings_logs_offline_errors(client: Client):
     """
-    :title: Check that messages with level 2 are logged when LDAP server doesn't exist
+    :title: Default debug_level logs offline errors
     :setup:
-        1. Set ldap_uri to a non-existing ldap-server
-        2. Start sssd with default debug level
-        3. Enable ifp responder
+        1. Configure SSSD with an invalid uri and enable ifp responder
+        2. Start SSSD with default debug level
+        3. Enable infopipe responder
     :steps:
         1. Check logs
         2. Check default domain status
     :expectedresults:
-        1. Domain logs should contain a log related to 'going offline'
-        2. LDAP is not connected
+        1. Logs contain connection errors
+        2. SSSD is not connected
     :customerscenario: True
     """
     client.sssd.domain["ldap_uri"] = "ldap://typo.invalid"
     client.sssd.enable_responder("ifp")
     client.sssd.start(debug_level=None, raise_on_error=False)
 
     logs = client.fs.read(client.sssd.logs.domain())
-    assert "Failed to connect, going offline" in logs, "String was not found in the logs"
+    assert "Failed to connect, going offline" in logs, "Offline error messages are not in logs!"
 
-    assert client.sssd.default_domain, "default_domain is None"
-    res = client.sssctl.domain_status(client.sssd.default_domain)
-    assert "LDAP: not connected" in res.stdout
+    assert client.sssd.default_domain is not None, "Failed to load default domain!"
+    result = client.sssctl.domain_status(client.sssd.default_domain)
+    assert result is not None
+    assert "LDAP: not connected" in result.stdout, "LDAP is connected!"
 
 
 @pytest.mark.ticket(bz=1915319)
 @pytest.mark.topology(KnownTopology.Client)
-def test_logging__default_debug_level_sbus(client: Client):
+def test_logging__default_settings_logs_sbus_remove_key_error(client: Client):
     """
-    :title: SBUS doesn't trigger failure message at modules startup
+    :title: Default debug_level logs sbus remove key error
```

Comment:
First, test actually checks that "does *not* log".

Seconds, I don't think that's worth automation at all, too narrow.
Maybe it would make more sense if it would check, for example, that there are no error messages at all (0x0010, 0x0020, 0x0040, 0x0080)
But check for a message specific to sbus impl (that very well could be changed meanwhile) isn't the best idea.

Reply:
I don't like these logging tests, but other people seem to want them. I'll happily drop this test.